docs/index.rst

```diff
@@ -70,8 +69,9 @@ Installation
 Mac OS X Install Notes
```

Comment:
thanks, @ghing. Let's be more aggressive and just completely remove this section from this page.

Reply:
Ok. Completely removed all references to special instructions in 5281661.